src/app/frontend/logs/logs_factory.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default class LogColorInversionService {
+  constructor() {
+    /** @export {boolean} inverted_. */
+    this.inverted_ = false;
+  }
+
+  getInverted() { return this.inverted_; }
+
+  invert() { this.inverted_ = !this.inverted; }
+}
+
+/**
+ * Service function for logs colors management.
+ * @return {!LogColorInversionService}
+ */
+export default function logsColorInversionService() {
```

Comment:
Hmm... You cannot have two export defaults. Remove this one. Maybe this causes the issue?